### PR TITLE
Update api-utils-python egg, bump some pypi dependencies (#25, #33)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR modifies `requirements.txt` so that the `api-utils-python` egg references a release (`v2.0`) instead of `master` and that we're using the latest low-risk release for the other `pypi` dependencies. Notably, I elected to postpone using more significant release of `Django` (`3.1`) and `mysqlclient` (`2.0.1`); see issue #33 for more details on my thinking on each of the `pypi` package changes. I also am fixing a sample command in the `README.md` for running the test suite with `coverage` and `virtualenv`. The PR aims to resolve issues #25 and #33.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ You can also set up the application using `virtualenv` by doing the following:
 To run the test suite, use the following commands:
 
 ```sh
-coverage manage.py test -v 3
+coverage run manage.py test -v 3
```
Comment (pushyamig) [inline README.md:153]: what was the reason for adding `run`?

Comment (ssciolla) [reply README.md:153]: I don't think it works if you don't have that. This is the command we are using in `start.sh`.
https://github.com/tl-its-umich-edu/placement-exams/blob/920c962d83b9473195b9b24d4c85f11a19c708c8/start.sh#L19-L22

Comment (pushyamig) [reply README.md:153]: OK, it's just a fix the doc with correct info then

Comment (ssciolla) [reply README.md:153]: Yup, I think I'll add `coverage report`, too.

**Comment by ssciolla** [comment]
Thanks for the reviews!

**Pull request merged**